ydb/public/lib/ydb_cli/dump/files/files.h

```diff
@@ -13,6 +13,7 @@ const TFileInfo& Changefeed();
 const TFileInfo& TopicDescription();
 const TFileInfo& CreateTopic();
 const TFileInfo& CreateCoordinationNode();
+const TFileInfo& CreateResource();
```

Comment:
Предлагаю сразу так:
```suggestion
const TFileInfo& CreateRateLimiterResource();
```

Мало ли какие ещё ресурсы появятся в будущем.